crates/ika-core/src/dwallet_mpc/crytographic_computation/mpc_computations/presign.rs

```diff
@@ -189,7 +193,21 @@ impl PresignPublicInputByProtocol {
                 let public_input =
                     <PresignParty<Secp256K1ECDSAProtocol> as mpc::Party>::PublicInput::from((
                         protocol_public_parameters,
-                        None,
+                        match dwallet_dkg_output {
+                            Some(dkg_output) => {
+                                let versioned_output = bcs::from_bytes::<
+                                    <Secp256K1ECDSAProtocol as dkg::Protocol>::DecentralizedPartyDKGOutput,
+                                >(&dkg_output)?;
+                                let output = match versioned_output {
+                                    VersionedOutput::TargetedPublicDKGOutput(output) => output,
+                                    VersionedOutput::UniversalPublicDKGOutput {
+                                        output, ..
+                                    } => output,
```

Comment:
In all of these cases (for all curves), we must reject the session on an error - only global presign is supported for universal dwallets. @ItayLevyOfficial